src/provider/koios.ts

```diff
@@ -0,0 +1,236 @@
+import {
+    Address,
+    Assets,
+    Credential,
+    Datum,
+    DatumHash,
+    Delegation,
+    Network,
+    OutRef,
+    ProtocolParameters,
+    Provider,
+    RewardAddress,
+    Transaction,
+    TxHash,
+    Unit,
+    UTxO,
+} from "../types/mod.ts";
+import {BackendFactory, KoiosHttpError, KoiosTimeoutError} from "@adabox/koios-ts-client/dist/index.js"
```

Comment:
Because it's Deno you can't import the library like this. Change the import to: `https://esm.sh/@adabox/koios-ts-client@1.0.6/dist/index.js`